keanu-project/src/test/java/io/improbable/keanu/vertices/intgr/nonprobabilistic/IntegerIfVertexTest.java

```diff
@@ -54,7 +52,15 @@ public void expectedValueMatchesBranchProbability() {
             .then(thenValue)
             .orElse(elseValue);
 
-        assertEquals(IntegerVertexTest.calculateMeanOfVertex(vertex), expectedMean, 0.1);
+        assertEquals(calculateMeanOfVertex(vertex), expectedMean, 0.01);
     }
 
+    private static double calculateMeanOfVertex(IntegerVertex vertex) {
+        BayesianNetwork net = new BayesianNetwork(vertex.getConnectedGraph());
+
+        return MetropolisHastings.withDefaultConfig(KeanuRandom.getDefaultRandom())
```

Comment:
If you use `withDefaultConfig()` it will use the default random anyway. So this parameter can be removed.